### PR TITLE
Ron/Setup: Wandb + Flink installer

### PR DESCRIPTION
**Pull request by ronyw7**
- Wandb logging added to `gen.py`. **To discuss**: what other params to log for experiments
- Flink installation script: `setup_flink.sh`. To use, download to local and run the script `./setup_flink.sh`. (Run `chmod +x setup_flink.sh` to ensure the script is executable.)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ logs
 target
 __pycache__
 *.log
+wandb/
```
Comment (franklsf95) [inline .gitignore:8]: add new line

File: nanobenchmarks/setup_flink.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Latest stable release
+FLINK_VERSION="1.18.0"
+
+# Create new conda env to install PyFlink
+conda create -n raydata-flink pip python=3.10
```
Comment (franklsf95) [inline nanobenchmarks/setup_flink.sh:7]: oh let's use the same raydata environment to install flink

File: nanobenchmarks/setup_flink.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# wget -c "$FLINK_DOWNLOAD_URL" -O "flink-$FLINK_VERSION.tgz"
+
+# echo "Unzipping Flink..."
+# tar -xzf "flink-$FLINK_VERSION.tgz" # -x extract; -z decompress with gzip; -f filename
+
+echo "Installing PyFlink..."
+pip install apache-flink==$FLINK_VERSION
+
+echo "Flink setup completed."
+echo "To start running flink, switch to the new conda environment:"
```
Comment (franklsf95) [inline nanobenchmarks/setup_flink.sh:27]: ditto, use the same environment

File: nanobenchmarks/gen.py [hunk 4]
```diff
@@ -38,13 +39,21 @@ def run_experiment(*, blowup: int = 0, parallelism: int = -1, size: int = -1):
...
+        "Array Size": ret
+        })
     return ret
 
 
 def main():
-    ray.init("auto")
     ray.data.DataContext.get_current().execution_options.verbose_progress = True
 
+    wandb.init(project='gen_spark', entity='ronyw') # change to entity='raysort'
```
Comment (franklsf95) [inline nanobenchmarks/gen.py:55]: Run the `black` formatter. Use double quotes.

**Review by franklsf95 — CHANGES_REQUESTED** [review]
See inline comments. Overall it looks good!

File: .gitignore (hunk 1 above)
Comment (ronyw7) [reply .gitignore]: Fixed and added this setting in black formatted! 

Quick note, this end-of-file newline doesn't appear to be showing on GitHub, e.g. https://github.com/franklsf95/ray-data-eval/blob/main/.gitignore

File: nanobenchmarks/setup_flink.sh (hunk 2 above)
Comment (ronyw7) [reply nanobenchmarks/setup_flink.sh]: Fixed.

File: nanobenchmarks/setup_flink.sh (hunk 3 above)
Comment (ronyw7) [reply nanobenchmarks/setup_flink.sh]: Fixed.

File: nanobenchmarks/gen.py (hunk 4 above)
Comment (ronyw7) [reply nanobenchmarks/gen.py]: Fixed, thanks Frank!

**Review by franklsf95 — APPROVED** [review]

**Pull request merged**